langchain/chains/llm_summarization_checker/memory.py

```diff
@@ -0,0 +1,27 @@
+"""Memory for summarization checker chain."""
+from typing import Any, Dict, List
+
+from pydantic import BaseModel
+
+from langchain.chains.base import Memory
+
+
+class ImmutableMemory(Memory, BaseModel):
```

Comment:
is this used anywhere?

Reply:
I was using it pass along the original summary text between multiple sequential chain runs without having to deal with input/output vars.   But that version of the prompt got dropped.

Thought it might still be useful so I kept it in, but happy to remove and open a different pull with some examples.

Reply:
yeah different pr probably best!

Reply:
Moved to #1293